src/lib/markdown/tables.ts

```diff
@@ -12,6 +12,24 @@ export default function markdownTables(md: MarkdownIt) {
         tokens[i].level--;
       }
 
+      // convert break line into br tag
```

Comment:
It seems there is a small bug here if you:

- add a newline into a table cell as the first item
- then add text on the second line
- reload the editor